app/stores/base/Store.ts

```diff
@@ -144,6 +149,42 @@ export default abstract class Store<T extends Model> {
     LifecycleManager.executeHooks(model.constructor, "afterRemove", model);
   }
 
+  @action
+  addToArchive(item: PartialWithArchivedAt<T>): void {
```

Comment:
```suggestion
  archive(item: PartialWithArchivedAt<T>): void {
```

I think just "archive" is good.

Reply:
Yeah, `archive` was already used, for example, in `DocumentsStore`, so I made it `addToArchive` for now.

Reply:
Yeah, but the separation is also to avoid the `documents.archive` call.

Reply:
I have felt this tension that it's unclear when a store method is working just on local data vs mutating the server too. Need to come up with a consistent pattern here 🤔

Reply:
Agreed! I'll give some thoughts...